### DISCUSSION TITLE
Update Geoiplookup Database

### DISCUSSION THREAD

**Discussion by ADadlDudlDa** [HomeITAdmin/nextcloud_geoblocker#120] [Q&A]
Hello Volks,

how can I update the Geoiplookup Database? Nextcloud says, that the database is from 2020-11-19?

Thank you 

ADadlDudlDa

**Comment by HomeITAdmin** [comment]
Hi,

you would update it in a similar way as you installed it, usually with the package manager of your distribution. 
But at least on Debian I think that is the newest version available. You would need to complain to the maintainer of the package.
This is definitely a disadvantage of this localization provider.

Best

**Comment by ADadlDudlDa** [comment]
Dear @HomeITAdmin ,

sorry, for answering late: Which provider are you using, and how can I find the maintainer of "Geoiplookup Database"

Thank you.

ADadlDudlDa

**Comment by HomeITAdmin** [comment]
Hi,

I personally use the RIR Data, but all of them have their advantages and disadvantages.
How to find the maintainer depends on you distro. I am also not an expert, but there is usually a way to report bugs or feature wishes.

Best
